lib/bundler/rubygems_integration.rb

```diff
@@ -461,8 +461,8 @@ def backport_spec_file
       end
     end
 
-    def redefine_method(klass, method, &block)
-      if klass.instance_methods(false).include?(method)
+    def redefine_method(klass, method, force_removal = false, &block)
+      if force_removal || klass.instance_methods.include?(method)
```

Comment:
It was intended to allow overriding the method check, as I believe this method wasn't being detected properly and thus not removed on windows, (generating the warning generated seen on our windows ci)